src/pretty-dom.js

```diff
@@ -1,21 +1,50 @@
 import prettyFormat from 'pretty-format'
+import {getDocument} from './helpers'
+
+function inCypress(dom) {
+  const window =
+    (dom.ownerDocument && dom.ownerDocument.defaultView) || undefined
+  return (
+    (typeof global !== 'undefined' && global.Cypress) ||
+    (typeof window !== 'undefined' && window.Cypress)
+  )
+}
+
+const inNode = () =>
+  typeof process !== 'undefined' &&
+  process.versions !== undefined &&
+  process.versions.node !== undefined
+
+const getMaxLength = dom =>
+  inCypress(dom) ? 0 : process.env.DEBUG_PRINT_LIMIT || 7000
 
 const {DOMElement, DOMCollection} = prettyFormat.plugins
 
-function prettyDOM(htmlElement, maxLength, options) {
-  if (htmlElement.documentElement) {
-    htmlElement = htmlElement.documentElement
+function prettyDOM(
+  dom = getDocument().body,
```

Comment:
If someone's not running in an environment where the document is globally available, then they have to provide a DOM node to log, otherwise we'll just default to `document.body` which I think is a good default. I imagine lots of people will just call `prettyDOM()` and that'll be helpful to them!